gf-webapp-admin/src/component/rooms/room/longDescription/longDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from "react";
+import {IRoom} from "../../../../model/room";
+import {saveRoom} from "../../../../service/room";
+import {EditBtn} from "../editBtn/EditBtn";
+
+export const LongDescription = ({room}: {room:IRoom}) => {
```

Comment:
add interface for component instead of props description

Reply:
Done